### DISCUSSION TITLE
Where to put toy examples?

### DISCUSSION THREAD

**Discussion by haipinglu** [pykale/pykale#317] [Ideas] [answered]
For toy examples like [this DA example](https://colab.research.google.com/github/sz144/pykale/blob/brain-example/examples/toy_domain_adaptation/tutorial.ipynb), where and how we should to place them? As an example like others? Or a dedicated ToyExamples/Tuturials under examples?

On the other hand, we should have more such toy examples to low the barriers.

**Comment by xianyuanliu** [comment, marked as answer]
We have put this toy example in [pykale/examples/toy_domain_adaptation](toy_domain_adaptation).
